common-jvm/build.gradle

```diff
@@ -4,33 +4,42 @@ plugins {
 }
 
 java {
-    sourceCompatibility = JavaVersion.VERSION_1_7
-    targetCompatibility = JavaVersion.VERSION_1_7
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
+    kotlinOptions {
+        jvmTarget = "1.8"
+        freeCompilerArgs += '-opt-in=kotlin.RequiresOptIn'
+    }
 }
 
 dependencies {
-    api('org.ergoplatform:ergo-appkit_2.11:develop-44fddd97-SNAPSHOT') {
+    api('org.ergoplatform:ergo-appkit_2.11:5.0.3') {
         exclude group: 'org.bouncycastle', module: 'bcprov-jdk15on'
         exclude group: 'org.bitbucket.inkytonik.kiama', module: 'kiama_2.11'
         exclude group: 'com.google.guava', module: 'guava'
     }
-    // kiama needs an own build for JRE7 compatibility
+    
+    // Custom Kiama - should work fine on Java 8
     api ('com.github.MrStahlfelge.kiama:core_2.11:2.1.1') {
         exclude group: 'com.google.guava', module: 'guava'
     }
     api ('com.github.MrStahlfelge.kiama:library_2.11:2.1.1') {
         exclude group: 'com.google.guava', module: 'guava'
     }
-    // stick to 20 for JRE7 compatibility
+    
+    // Guava 20 works with Java 8 (could upgrade later if needed)
```

Comment:
Same: all the excludes and redefinitions are not needed on Java 8. But, probably we will loose ios.